botbot/apps/plugins/core/logger.py

```diff
@@ -10,6 +12,12 @@ class Config(config.BaseConfig):
         help_text="Don't log lines starting with this string"
     )
 
+def should_ignore_text(text, ignore_prefix):
+    return any(
+        (re.match(prefix, text, flags=re.IGNORECASE) is not None)
```

Comment:
I figured that this would allow people to set more complex regular expressions if needed. I'd guess `text.lower().startswith(prefix.lower())` is more efficient, since re.match will compile prefix into a regular expression each time, but since the rate at which chat messages are coming in is so low, it's not an issue.